README.md

```diff
@@ -28,150 +20,67 @@ There are two examples, which may be used as reference:
 Requirements
```

Comment:
Can't add this note on the right lines. I think that the links above should be updated.

Reply:
Ok, I added the second animation and added the links on README.